### DISCUSSION TITLE
UN SDG Indicators Database

### DISCUSSION THREAD

**Discussion by nikolamedi** [transport-data/community#27] [Sources]
UN SDG Indicators Database has several data points:

- Indicator 3.6.1, Series : Death rate due to road traffic injuries, by sex (per 100,000 population) SH_STA_TRAF (based on WHO)
- Indicator 9.1.2, Series : Freight volume, by mode of transport (tonne kilometres) IS_RDP_FRGVOL (based on ICAO, ITF)
- Indicator 9.1.2, Series : Passenger volume (passenger kilometres), by mode of transport IS_RDP_PFVOL (based on ICAO, ITF)
- Indicator 9.1.2, Series : Container port traffic, maritime transport (twenty-foot equivalent units - TEUs) IS_RDP_PORFVOL (based on UNCTAD)
- Indicator 9.1.2, Series : Freight loaded and unloaded, maritime transport (metric tons) IS_RDP_LULFRG (based on UNCTAD)
- Indicator 11.2.1, Series : Proportion of population that has convenient access to public transport (%) SP_TRN_PUBL (based on UNHABITAT)

https://unstats.un.org/sdgs/dataportal/database

**Comment by khaeru** [comment]
Thanks, @nikolamedi! This is a good one to tap into early, because they are already using SDMX under the hood. That means it should be more or less directly interoperable with TDC data and metadata.
